class-tgm-plugin-activation.php

```diff
@@ -249,7 +249,7 @@ public function init() {
 			// After this point, the plugins should be registered and the configuration set.
 
 			// Proceed only if we have plugins to handle.
-			if ( $this->plugins ) {
+			if ( ! empty( $this->plugins ) ) {
 				$sorted = array();
```

Comment:
As long as you first check for `is_array()`, it doesn't matter whether you use `$var !== array()`, `! empty( $var )` or `count( $var ) > 0` for the result.

Using `count()` is definitely slowest, so either of the first two is better. I seemed to remember that `!==array()` was slightly faster (might depend on the PHP version), but am happy to accept `empty()` just as well.